src/store/lib/lookup-tables.ts

```diff
@@ -371,6 +371,7 @@ export function createRelations(metadata: Metadata, tables: LookupTables) {
     // Index multi-class investigators.
     if (
       card.type_code === "investigator" &&
+      (card.deck_limit || 0) > 0 &&
```

Comment:
This makes sense. 👍 We can make it a bit shorter given we only care about positive numbers:
```suggestion
      card.deck_limit &&
```